### PR TITLE
adding link residence to death page

### PR DESCRIPTION
**Pull request by rubyimrie**
addding link page, no big changes

### CODE REVIEW

File: pages/link-residence-death.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">Linking Residence to Place of Death</h1>
```
Comment (Sephster) [inline pages/link-residence-death.vue]: Will we need these h1 styles inline once we've set them globally? I'm thinking we might be able to remove these. Thanks Ruby

File: pages/link-residence-death.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+ export default {
+   data: () => ({
+
+   }),
+   methods: {
+     showPageInfo() {
+       Swal.fire({
+         title: 'Linking Residence to Place of Death',
+         html: '<div>This map shows the geographical relationship between an accused witch’s place of residence and place of death. There are only 111 places of death recorded within the dataset. When selecting a coloured line, the recorded  <b class="font-bold">first location</b> is where the <b class="font-bold">person died, second location</b> is where the <b class="font-bold">accused witch resided</b> and <b class="font-bold">below</b> these places is the <b class="font-bold">name</b> of the accused witch </div>',
```
Comment (Sephster) [inline pages/link-residence-death.vue]:
```suggestion
         html: '<div>This map shows the geographical relationship between an accused witch’s place of residence and place of death. There are only 111 places of death recorded within the dataset. When selecting a coloured line, the recorded  <strong>first location</strong> is where the <strong>person died, second location</strong> is where the <strong>accused witch resided</strong> and <strong>below</strong> these places is the <strong>name</strong> of the accused witch </div>',
```

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/link-residence-death.vue (hunk 1 above)
Comment (rubyimrie) [reply pages/link-residence-death.vue]: yeah can remove them, done

File: pages/link-residence-death.vue (hunk 2 above)
Comment (rubyimrie) [reply pages/link-residence-death.vue]: done

**Pull request merged**